src/main/java/com/exasol/errorcodecrawlermavenplugin/model/ExasolError.java

```diff
@@ -0,0 +1,95 @@
+package com.exasol.errorcodecrawlermavenplugin.model;
+
+public class ExasolError {
```

Comment:
I find the name a bit misleading, as it makes me think that there was an error situation, but this actually represents something more like an error message declaration/invocation.

Would make sense to rename it, eg, `ExasolErrorMessage`, `ErrorMessageInvocation` or  `ErrorMessageDeclaration` or similar?